### ISSUE TITLE
"Show user tweets only" in the timeline

### ISSUE THREAD

**Issue by m3k1e** [Bl4Cc4t/GoodTwitter2#512]
Hey there, I saw a new feature from another [old twitter layout](https://github.com/dimdenGD/OldTwitter) (Non-userscript) where you can toggle off retweets to see people you're following in the timeline. It would be cool if you added it anyway.

Source: https://twitter.com/dimdenEFF/status/1604826353072758784
![image](https://user-images.githubusercontent.com/41316133/209286154-79c437ee-cee4-40e2-87dd-c9e781fd3fb9.png)

**Comment by Bl4Cc4t** [comment]
I plan to add something like this when my rewrite of the script is in a usable state.
thanks for suggesting!

also, it's nice to see other projects like the one you linked :)

**Comment by MattPD** [comment]
Somewhat related, would it be possible to mute "Quote Tweets" (similar intent to "show user tweets only" but at a finer granularity?). FWIW, TweetDeck allows to mute Quote Tweets from a specific user (so there may be a way to filter these), although an option to just do it globally would work just fine for me, too.

**Comment by Bl4Cc4t** [comment]
yes, that should be possible. but like the other feature here, only in the new version I'm working on :)
